runtime/src/protocol.rs

```diff
@@ -206,11 +206,7 @@ impl Protocol {
                         // is no need to do anything more.
                         return Ok(());
                     }
-                    Err(error) => Body::Error {
-                        module: "".to_owned(), // XXX: Error codes.
-                        code: 1,               // XXX: Error codes.
-                        message: format!("{}", error),
-                    },
+                    Err(error) => Body::Error(Error::new("dispatcher", 1, &format!("{}", error))),
```

Comment:
could you add a note for developers on the change in style in protocol.rs

Reply:
See above comment, will add a changelog entry for this part.